docs/aca/09-aca-autoscale-keda/index.md

```diff
@@ -51,26 +54,25 @@ triggers:
+* The property `messageCount` is used to decide when scaling out should be triggered. In our case, it will be set to `10`.
 * The property `cloud` represents the name of the cloud environment that the service bus belongs to.
 
-{: .note }
-Note about authentication: KEDA scaler for Azure Service Bus supports different authentication mechanisms such as [Pod Managed Identity](https://learn.microsoft.com/en-us/azure/aks/use-azure-ad-pod-identity), [Azure AD Workload Identity](https://azure.github.io/azure-workload-identity/docs/), and shared access policy (connection string). When using KEDA with Azure Container Apps, at the time of writing this module, the only supported authentication mechanism is Connection Strings. on ACA roadmap, there is a work item on ACA product backlog to enable [KEDA Scale with Managed Identity.](https://github.com/microsoft/azure-container-apps/issues/592)
+!!! note
+    Note about authentication: KEDA scaler for Azure Service Bus supports different authentication mechanisms such as [Pod Managed Identity](https://learn.microsoft.com/en-us/azure/aks/use-azure-ad-pod-identity), [Azure AD Workload Identity](https://azure.github.io/azure-workload-identity/docs/), and shared access policy (connection string). At the time of writing this workshop, when using KEDA with Azure Container Apps the only supported authentication mechanism is Connection Strings. There is a work item in the ACA product backlog that involves enabling [KEDA Scale with Managed Identity.](https://github.com/microsoft/azure-container-apps/issues/592)
 
-Azure Container Apps has its own proprietary schema to map KEDA Scaler template to its own when defining a custom scale rule, you can define this scaling rule via Container Apps [ARM templates](https://learn.microsoft.com/en-us/azure/container-apps/azure-resource-manager-api-spec?tabs=arm-template#container-app-examples), [yaml manifest](https://learn.microsoft.com/en-us/azure/container-apps/azure-resource-manager-api-spec?tabs=arm-template#container-app-examples), Azure CLI, or from Azure Portal. In this module, we will cover how to do it from the Azure CLI.
+Azure Container Apps has its own proprietary schema to map KEDA Scaler template to its own when defining a custom scale rule. You can define this scaling rule via Container Apps [ARM templates](https://learn.microsoft.com/en-us/azure/container-apps/azure-resource-manager-api-spec?tabs=arm-template#container-app-examples), [yaml manifest](https://learn.microsoft.com/en-us/azure/container-apps/azure-resource-manager-api-spec?tabs=arm-template#container-app-examples), Azure CLI, or from the Azure Portal. In this module, we will cover how to do it from the Azure CLI.
 
-##### 1. Create a new secret in Container App
+##### 1. Create a New Secret In The Container App
```

Comment:
One less #, so not everything is in caps like other pages ? and same for other headings down below as well.